pages/admin/badges-edit.js

```diff
@@ -502,7 +503,7 @@ export class AdminBadgesEdit extends Component {
   }
 }
 
-const Page = connect(['authenticatedUser', 'error', 'badge', 'admin'], actions)(AdminBadgesEdit)
+const Page = connect(['authenticatedUser', 'error', 'badge', 'admin', 'notification'], actions)(AdminBadgesEdit)
```

Comment:
```suggestion
const Page = connect(['authenticatedUser', 'error', 'badge', 'admin'], actions)(AdminBadgesEdit)
```

The 'notification' property of state isn't needed.